src/types/lib/ConsiderationConstants.sol

```diff
@@ -334,9 +334,9 @@ uint256 constant OrderFulfilled_post_memory_region_reservedBytes = 0x60;
  * ...
  *
  * Note that the padded calldata selector will be at minimum at the
- * 0x80 memory slot if no total original additional recipients were provided
+ * 0x80 memory slot.
  */
-uint256 constant authorizeOrder_calldata_baseOffset = 0x80; // 0x200 - 0x180
+uint256 constant authorizeOrder_calldata_baseOffset = OrderFulfilled_offer_length_baseOffset - 0x180;
```

Comment:
This might be nicer. But I think in general in this file, the authors have been just inling the final value and added comments have that final value was derived.

cc: @0age